### DISCUSSION TITLE
WinMergeU - CLI - Report with Diff Context

### DISCUSSION THREAD

**Discussion by Derek-Hartman** [WinMerge/winmerge#2117] [Q&A] [answered]
I saw in another discussion about what the command line in for generating a report which works great. 
`"c:\Program Files\WinMerge\WinMergeU.exe" -noninteractive file1\t1.txt  file2\t1.txt -or c:\tmp\report.html"`
But now I am wondering if it is possible to set the Diff Context to that generated report?

**Comment by sdottaka** [comment, marked as answer]
The Diff Context set in the View → Diff Context menu also applies to reports.
To set the Diff Context using a command line argument, specify `/cfg DiffContextV2=` as shown below.

~~~
"c:\Program Files\winmerge\WinMergeU.exe" file1 file2 /or c:\tmp\report.html /noninteractive /cfg DiffContextV2=0
~~~

**Comment by phreeplot** [comment]
That is great thanks.
